openmapflow/constants.py

```diff
@@ -13,7 +13,7 @@
 TEMPLATE_README = TEMPLATES_DIR / "README.md"
 TEMPLATE_DEPLOY_YML = TEMPLATES_DIR / "github-deploy.yaml"
 TEMPLATE_TEST_YML = TEMPLATES_DIR / "github-test.yaml"
-VERSION = "0.1.6rc4"
+VERSION = "0.2.0rc1"
```

Comment:
This is a breaking change but I am only updating the minor version since the package is still more or less under development